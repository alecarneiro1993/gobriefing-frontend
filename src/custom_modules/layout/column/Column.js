import React from "react";
import PropTypes from "prop-types";
import Grid from "@material-ui/core/Grid";

export default function Column({ children, ...rest }) {
  return (
    <Grid item {...rest}>
      {children}
    </Grid>
  );
}

Column.propTypes = {
  children: PropTypes.oneOfType([
    PropTypes.arrayOf(PropTypes.node),
    PropTypes.node
  ]).isRequired
};
