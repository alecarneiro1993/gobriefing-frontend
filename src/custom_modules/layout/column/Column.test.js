import React from "react";
import { shallow } from "enzyme";

import Column from ".";

describe("Column", () => {
  it("renders without crashing", () => {
    const wrapper = shallow(<Column>Content</Column>);
    expect(wrapper).toMatchSnapshot();
  });
});
