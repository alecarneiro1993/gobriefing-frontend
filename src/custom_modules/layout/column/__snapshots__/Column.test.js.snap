// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Column renders without crashing 1`] = `
<WithStyles(ForwardRef(Grid))
  item={true}
>
  Content
</WithStyles(ForwardRef(Grid))>
`;
